palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/web/ResourceClient.java

```diff
@@ -48,4 +50,7 @@
     @PostMapping(path = "/addResource", consumes = "application/json", produces = "application/json")
     Boolean addResource(final URI url, @RequestBody final AddResourceRequest request);
 
+    @GetMapping(path = "/actuator/health", produces = "application/*+json")
```

Comment:
Why is this one json-like but the rest were json?

Reply:
reverted